### PR TITLE
chore: Updates release related assets

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
Following the [Release Process changes discussion](https://github.com/apache/superset/discussions/27769), this PR:
- Adds instructions to report security issues
- Changes the voting template to make it clear that voting rounds should be opened for at least 72 hours

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ body:
       value: |
         Hello Superset Community member! Please keep things tidy by putting your post in the proper place:
 
+        🚨 Reporting a security issue: send an email to security@apache.org. DO NOT USE GITHUB ISSUES TO REPORT SECURITY PROBLEMS.
```
Comment (rusackas) [inline .github/ISSUE_TEMPLATE/bug-report.yml:10]: Is there a way to make that last sentence in a 256pt blinking red font?

File: RELEASING/email_templates/vote_pmc.j2 [hunk 2]
```diff
@@ -39,7 +39,7 @@ https://github.com/apache/{{ project_module }}/blob/{{ version_rc }}/UPDATING.md
 Public keys are available at:
 https://www.apache.org/dist/{{ project_module }}/KEYS
 
-The vote will be open for at least 72 hours or until the necessary number
+The vote will be open for at least 72 hours and until the necessary number
```
Comment (rusackas) [inline RELEASING/email_templates/vote_pmc.j2:42]: Maybe "The vote will be left open until at least 72 hours have passed and the necessary number of votes (3) have been reached."

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -46,7 +47,6 @@ body:
       options:
         - master / latest-dev
         - "4.0.0"
-        - "3.1.2"
```
Comment (rusackas) [inline .github/ISSUE_TEMPLATE/bug-report.yml:49]: If 3.1 is the last minor of our two supported versions, shouldn't we continue to collect bugs for it? They might be critical in some cases, or point to issues that are also in 4.0

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (dpgaspar) [reply .github/ISSUE_TEMPLATE/bug-report.yml:10]: let's change the email to security@superset.apache.org, inline with: https://github.com/apache/superset/security/policy

**Review by rusackas — APPROVED** [review]

**Pull request merged**